source/scripts/Background/index.ts

```diff
@@ -48,6 +48,7 @@ browser.runtime.onMessage.addListener(async ({ type, target }) => {
 });
 
 browser.runtime.onConnect.addListener(async (port: Runtime.Port) => {
+  if (port.name === 'pali') window.controller.wallet.setIsPopupOpen(true);
```

Comment:
Is port.name === 'pali' only when a dApp calls a method through pali contentScripts and all the times a popUp is opened externally?

Reply:
port.name === 'pali' is when user open pali manually or when pali is opened externally by dapps